### ISSUE TITLE
User setting not working for markdown file

### ISSUE THREAD

**Issue by parthi2929** [Microsoft/vscode#67743]
When I generically set below setting to disable annoying intellisense while writing markdown document (for every letter, there is a suggestion!), it works. That is, intellisense is no more working for any file. 

```
        "editor.quickSuggestions": {
            "other": false,
            "comments": false,
            "strings": false
        }
```

But I want to set this only for markdown files, in that case the below code is simply not working. The intellisense auto suggestions still popup. 

```
        "[markdown]": {"editor.quickSuggestions": {
            "other": false,
            "comments": false,
            "strings": false
        }},
```

- VSCode Version: 1.30.2
- OS Version:Win 10

Steps to Reproduce:

1.  Go to user setting and enter above snippet as explained.
2. Open any markdown file and start typing. You will observe quick suggestions, when you specifically disabled for markdown, but not when you generically disable for all files. 

Does this issue occur when all extensions are disabled?: No. No suggestions happen if I enable or disable quicksuggestions at all.

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all extensions disabled?

**Comment by parthi2929** [comment]
I did not try insiders build, but I think it could have been due to a silly bug from my side. I had the file named as "filename. md" with a space. This was recognized as markdown ([MPE](https://shd101wyy.github.io/markdown-preview-enhanced/#/) preview was happening so did not realize), with no error. Once I made "filename.md", things worked. So I guess this should be closed, if VS was not supposed to take "filename. md" as markdown.
